src/pixiewps-wrapper.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+
+aaaa=$(date +%Y)
+mm=$(date +%m)
+mm=$(($mm-2))
+if [ $mm -le 0 ] ; then
+	mm=$(($mm+12))
+	aaaa=$(($aaaa-1))
+fi
+
+if [ $mm -lt 10 ] ; then
+	mm='0'$mm
+fi
+
+start=$mm\/$aaaa
+
+mm=$(($mm+3))
+if [ $mm -gt 12 ] ; then
+	mm=$(($mm-12))
+	aaaa=$(($aaaa+1))
+fi
+
+if [ $mm -lt 10 ] ; then
+	mm='0'$mm
+fi
+
+end=$mm\/$aaaa
+
+intervals=$start:$end'
+01/1970:03/1970
+01/2015:03/2015
+01/2000:03/2000
+01/2013:03/2013
+'
+
+for i in $intervals ; do
+	start=$(printf "%s\n" "$i" | cut -d : -f 1)
+	end=$(printf "%s\n" "$i" | cut -d : -f 2)
+	# Run pixiewps with optimization
+	if pixiewps $@ --start $start --end $end > tmp.$$$ ; then
+		cat tmp.$$$ | grep 'WPS pin:' | cut -d ':' -f 2 | sed 's/^[ ]\+//'
+		rm tmp.$$$
+		return 0
+	elif cat tmp.$$$ | grep -o 'vulnerable' ; then
+		rm tmp.$$$
+		continue
+	else
+		rm tmp.$$$
+		return 1
+	fi
+done
+
+# Reruns Pixiewps with -f/--force option
+echo "[?] Do you want to reruns Pixiewps with --force option? [n/Y]"
+read answer;
```

Comment:
what's the point of asking interactive questions? wasn't the entire purpose of calling pixiewps from reaver to do everything automatically ?

Reply:
Ok, do everything automatically.